docs/how_to/single_node.md.source

```diff
@@ -0,0 +1,144 @@
+# M3DB Single Node Deployment
+
+Deploying a single-node cluster is a great way to experiment with M3DB and get a feel for what it
+has to offer. Our Docker image by default configures a single M3DB instance as one binary
+containing:
+
+- An M3DB storage instance (`m3dbnode`) for timeseries storage. This includes an embedded tag-based
+  metrics index, as well as as an embedded etcd server for storing the above mentioned cluster
+  topology and runtime configuration.
+- A "coordinator" instance (`m3coordinator`) for writing and querying tagged metrics, as well as
+  managing cluster topology and runtime configuration.
+
+To begin, first start up a Docker container with port `7201` (used to manage the cluster topology)
+and port `9003` (used to read and write metrics) exposed. We recommend you create a persistent data
+directory on your host for durability:
+
+```
+docker pull quay.io/m3/m3dbnode:latest
+docker run -p 7201:7201 -p 9003:9003 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db quay.io/m3/m3dbnode:latest
+```
+<operation>
+<validation>
+docker pull quay.io/m3/m3dbnode:latest
+docker create --name "${CONTAINER_NAME}" -p 7201:7201 -p 9003:9003 -p 9002:9002 quay.io/m3/m3dbnode:latest
+docker start "${CONTAINER_NAME}"
+if [ $? -ne 0 ]; then
+  echo "m3dbnode docker failed to start"
+  docker logs "${CONTAINER_NAME}"
+  exit 1
+fi
+
+sleep 5
+echo "checking uptime of coordinator service"
+ATTEMPTS=6 TIMEOUT=1 retry_with_backoff  \
+  '[ "$(curl -sSf 0.0.0.0:7201/health | jq .uptime)" ]'
+</validation></operation>
+
+<!-- TODO: link to docs containing explanations of what namespaces, the coordinator,
```

Comment:
while you're here, mind adding links to this (we added recently): 
https://m3db.github.io/m3/operational_guide/placement/
https://m3db.github.io/m3/operational_guide/namespace_configuration/